commands/cluster_commands/install

```diff
@@ -325,7 +325,12 @@ helm_deploy() {
     else 
       echo "${i} Failed to Installed"
       status_log $TYPE_ERROR "${i} Failed to Installed due to following error"
-      eval "$(getGumSpinnerOrLogger "Installing ${i} component") helm upgrade --install $i oci://public.ecr.aws/${awsregistry}/$i -n ${NS} ${version} --create-namespace -f /tmp/values-override.yaml $values_yaml_file_names"  >&2    
+
+      if [ "${TARGET_PLATFORM}" != $Kubefirst ]; then
```

Comment:
we can remove this check.
and simplify the logic
by adding a check above life following
if [ "${TARGET_PLATFORM}" == $Kubefirst ]; then
   values_yaml_file_names="$values_yaml_file_names -f ${GRPL_WORKDIR}/template-files/values-kubefirst.yaml" 
fi

I'll do that

Reply:
oh I saw you already did something slimier here

https://github.com/grapple-solutions/grapple-cli/pull/90/files

Reply:
yes, sorry, I am always to fast with pushing my code :-(
especially in this case, as I had to build a docker image (which requires a new grpl cli version) before I could test it...
And I fully agree with your input.